apps/angular/src/app/features/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, inject } from '@angular/core';
+import { MatTableModule } from '@angular/material/table';
+import { AnimeApiService } from '@js-camp/angular/core/services/anime-api.service';
+import { AsyncPipe, DatePipe } from '@angular/common';
+import { EmptyPipe } from '@js-camp/angular/shared/pipes/empty.pipe';
+import { ProgressBarComponent } from '@js-camp/angular/shared/components/progress-bar/progress-bar.component';
+
+/** Column headers to be displayed in table. */
+export enum ColumnsHeaders {
+	Image = 'Image',
+	EnglishTitle = 'English title',
+	JapaneseTitle = 'Japanese title',
+	AiredStart = 'Aired starts with',
+	Type = 'type',
+	Status = 'status',
+}
+
+/** Dashboard component. Contains table with list of anime. */
+@Component({
+	selector: 'camp-dashboard',
+	standalone: true,
+	templateUrl: './dashboard.component.html',
+	styleUrl: './dashboard.component.css',
+	imports: [
+		MatTableModule,
+		AsyncPipe,
+		DatePipe,
+		EmptyPipe,
+		ProgressBarComponent,
+	],
+})
+export class DashboardComponent {
+
+	private readonly animeApiService = inject(AnimeApiService);
+
+	/** Stream of anime. */
+	protected readonly anime$ = this.animeApiService.getList();
```

Comment:
Please review [this practice](https://wiki.saritasa.rocks/frontend/frameworks/angular/best-practices#do-not-subscribe-inside-a-constructor) from our wiki, `async pipe` section and it's look something like this
```suggestion
	protected readonly animeList$: Observable<Anime[]>;

	public constructor() {
		this.animeList$ = this.animeApiService.getList();
	}
```

Reply:
Hmm I don't know why this approach is outdated, the [angular docs](https://angular.dev/guide/pipes/unwrapping-data-observables) still use it to demo. I think this just personal choice, pick the one that suitable for you.

Reply:
Okay, thanks for the reply!